libs/ledger-live-common/src/families/near/specs.ts

```diff
@@ -0,0 +1,132 @@
+import invariant from "invariant";
+import { BigNumber } from "bignumber.js";
+import type { Transaction, NearAccount } from "./types";
+import { getCryptoCurrencyById, parseCurrencyUnit } from "../../currencies";
+import { pickSiblings } from "../../bot/specs";
+import type { AppSpec } from "../../bot/types";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { acceptTransaction } from "./speculos-deviceActions";
+
+const currency = getCryptoCurrencyById("near");
+const minimalAmount = parseCurrencyUnit(currency.units[0], "0.00001");
+const stakingFee = parseCurrencyUnit(currency.units[0], "0.002");
+const maxAccount = 3;
+const validator = "ledgerbyfigment.poolv1.near";
+
+const near: AppSpec<Transaction> = {
+  name: "NEAR",
+  currency,
+  appQuery: {
+    model: DeviceModelId.nanoS,
+    appName: "NEAR",
+  },
+  genericDeviceAction: acceptTransaction,
+  testTimeout: 2 * 60 * 1000,
+  mutations: [
+    {
+      name: "Move 50% to another account",
+      maxRun: 1,
+      transaction: ({ account, siblings, bridge, maxSpendable }) => {
+        invariant(maxSpendable.gt(minimalAmount), "balance is too low");
+        const sibling = pickSiblings(siblings, maxAccount);
+        const recipient = sibling.freshAddress;
+        const amount = maxSpendable.div(2).integerValue();
+        return {
+          transaction: bridge.createTransaction(account),
+          updates: [{ recipient }, { amount }],
+        };
+      },
+    },
```

Comment:
please try to add a `test` clause to effectively cover testing the "effect" of the mutation on the account. you have example of this in other coins, for instance tron, ethereum,..

Reply:
Addressed here: https://github.com/LedgerHQ/ledger-live/pull/1126/commits/55445d90c377bd85b635f4d28588908f41391d91